K2Bridge/Factories/AggregateFactory.cs

```diff
@@ -167,57 +167,57 @@ private static ExtendedStatsAggregate GetExtendedStatsAggregate(string columnNam
             var jObject = (JObject)row[columnName];
 
             var count = jObject[AggregationsConstants.Count].Value<long?>();
```

Comment:
Since you are nullchecking anyway, you can have:
`jObject[AggregationsConstants.Count]?.Value<long?>();` and avoid a potential NRE (and for the others too)

Reply:
good catch !

Reply:
I think that because of that you can go back to ?? before?

Reply:
No, Look my screenshot on the previous comment.
Even using a `double?` count will not allow us to use `??`

Reply:
Ok I see, I'll mark this as resolved and continue in the other one